sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs

```diff
@@ -19,13 +19,13 @@ namespace Azure.Core.TestFramework
     /// </summary>
     public abstract class TestEnvironment
     {
-        private static readonly string RepositoryRoot;
+        public static string RepositoryRoot { get; }
```

Comment:
Keep this internal.

Reply:
It's needed from Azure.Core.Tests. We don't have IVT enabled.

Reply:
Will enable IVT.

Reply:
Tried enabling but there are conflicts between shared types from Azure.Core and TestFramework.